### PR TITLE
Improvement

### PR DESCRIPTION
**Pull request by chloeleichen**
1. show error message for featured dataset (https://github.com/TerriaJS/magda/issues/139). This happens because the featured dataset uses promises.all, and we need to make the datasets show error independently
2. Fix an error in record parsing

### CODE REVIEW

**Comment by steve9164** [comment]
I added another commit fixing another couple of places (that I originally modified) that were wrong.

File: magda-web-client/src/config.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ export const config = {
   appTitle: 'Australian open data search',
   featuredDatasets: [
     "19432f89-dc3a-4ef3-b943-5326ef1dbecc",
-    "bdcf5b09-89bc-47ec-9281-6b8e9ee147aa"
+    "0ff854cc-a621-4929-89d3-f0370c3dd344"
```
Comment (kring) [inline magda-web-client/src/config.js]: Did you mean to commit this?

File: magda-web-client/src/helpers/publisher.js [hunk 2]
```diff
@@ -17,11 +26,16 @@ type PublisherRaw = {
 
 
 export function parsePublisher(publisherRaw: PublisherRaw) : Publisher{
+    let error = null;
+    if(publisherRaw && !publisherRaw.id){
+      error = publisherRaw.message || 'an error occuered';
```
Comment (kring) [inline magda-web-client/src/helpers/publisher.js]: typo, occuered -> occurred

File: magda-web-client/src/config.js (hunk 1 above)
Comment (chloeleichen) [inline magda-web-client/src/config.js]: nope

**Comment by kring** [comment]
looks good other than those two small things.

**Pull request merged**